src/amoc_user.erl

```diff
@@ -9,6 +9,10 @@
 -define(REPEAT_INTERVAL,
     application:get_env(amoc, repeat_interval, ?REPEAT_INTERVAL_DEFAULT)).
 
+-define(REPEAT_NUM_DEFAULT, infinity).
+-define(REPEAT_NUM,
+    application:get_env(amoc, repeat_num, ?REPEAT_NUM_DEFAULT)).
```

Comment:
Wouldn't a function definition (e.g. ``repeat_num/0``) be better for this purpose?
Yes, the repeat interval value is defined with macros as well, but I believe it also should be changed.